README.md

```diff
@@ -248,10 +255,10 @@ npm install --add redux-time
 
 Then add this to your page's entry point, next to `createStore`:
 ```javascript
-import {animations, startAnimation} from 'redux-time'
+import {animationsReducer, startAnimation} from 'redux-time'
 
 // add animations to your reducers
-window.store = createStore(combineReducers({..., animations}))  
+window.store = createStore(combineReducers({..., animations: animationsReducer}))  
```

Comment:
animationsReducer -> animations.

You can define it as `animationsReducer` internally, but please export `animations`:

```javascript
exports = {animations: animationsReducer, ...}
```

Reply:
How does that solve the ambiguity problem? It's not ambiguous at the site of definition, it's ambiguous when it's used.